tools/psm_validation/macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@VERSION@">1.0.3</token>
+    <token name="@VERSION@">1.0.3+galaxy1</token>
```

Comment:
can you move the galaxy1 to the individual tool? See here for recommendation: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

The idea is that we can bump tool versions automatically in the future :)

Reply:
Will do.

Reply:
Added suffix as well:

`<token name="@VERSION_SUFFIX@">1</token>`
`<tool id="psmvalidator" name="PSM-Fragment Validator" version="@VERSION@+galaxy@VERSION_SUFFIX@">`